src/spring-cloud/azext_spring_cloud/custom.py

```diff
@@ -60,16 +59,23 @@ def spring_cloud_create(cmd, client, resource_group, name, location=None, app_in
             service_runtime_network_resource_group=service_runtime_network_resource_group
         )
 
-    if sku is None:
-        sku = "Standard"
     full_sku = models.Sku(name=_get_sku_name(sku), tier=sku)
 
-    check_tracing_parameters(app_insights_key, app_insights, disable_distributed_tracing)
-    update_tracing_config(cmd, resource_group, name, location, properties,
-                          app_insights_key, app_insights, disable_distributed_tracing)
-
     resource = models.ServiceResource(location=location, sku=full_sku, properties=properties, tags=tags)
-    return sdk_no_wait(no_wait, client.create_or_update, resource_group_name=resource_group, service_name=name, resource=resource)
+
+    poller = client.services.create_or_update(
+        resource_group, name, resource)
+    logger.warning(" - Creating Service ..")
+    while poller.done() is False:
+        sleep(5)
```

Comment:
Is it possible that in case of some failure, the returned response is not a poller?